src/game_inventory.cpp

```diff
@@ -1311,6 +1311,10 @@ class weapon_inventory_preset: public inventory_selector_preset
             }, _( "WIELD COST" ) );
         }
 
+        bool is_shown( const item_location &loc ) const override {
+            return !loc->made_of( phase_id::LIQUID );
```

Comment:
There are gas and plasma that should not be wielded too, so maybe it is better to check for `SOLID` instead?

Reply:
Thank you for the suggestion! Updated to list `made_of( phase_id::SOLID )` items only.